autopilot/src/autopilot.cpp

```diff
@@ -87,14 +88,23 @@ AutoPilotNode::AutoPilotNode(ros::NodeHandle& node_handle) : nh(node_handle)
   nh.param<double>("/autopilot_node/depth_imin", depthIMin, 0.0);
   nh.param<double>("/autopilot_node/depth_d", depthDGain, 0.0);
 
+  nh.param<double>("/autopilot_node/altitude_p", altitudePGain, 1.0);
+  nh.param<double>("/autopilot_node/altitude_i", altitudeIGain, 0.0);
+  nh.param<double>("/autopilot_node/altitude_imax", altitudeIMax, 0.0);
+  nh.param<double>("/autopilot_node/altitude_imin", altitudeIMin, 0.0);
+  nh.param<double>("/autopilot_node/altitude_d", altitudeDGain, 0.0);
+  nh.param<double>("/autopilot_node/max_altitude_command",maxAltitudeCommand, 25.0);
```

Comment:
@patapufate why stating the default value in two places? Also, where did that 25 m cap came from?